common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -132,6 +234,12 @@ case class WidthsByBreakpoint(
 
     case _ => Seq.empty
   }
+
+  val profiles: Seq[Profile] = allWidths
```

Comment:
We are pretty close to merging `Profile` and `BrowserWidth` together. Fist bump!